seeder/postgres_seeder/plugin.js

```diff
@@ -0,0 +1,140 @@
+"use strict";
+
+import Controller from "../../src/controller/index";
+
+export default class Seeder {
+  constructor(config) {
+    this.controller = new Controller();
+
+    this.productRange = config.product_range;
+    this.styleRange = config.style_range;
+    this.womenSizes = config.women_sizes;
+    this.menSizes = config.men_sizes;
+  }
+
+  async seed() {
+    const documents = await this.generateData();
+
+    for (let i = 0; i < documents.length; i++) {
+      this.controller.write(documents[i]).catch(function (err) {
+        return console.error(`Error: ${err}`);
```

Comment:
**Annotation:** use return to stop for loop if an error occurs with the write method.